pages/verify-certification-score.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <div class="page competences">
+    <main class="page-body">
+      <div class="container lg certificate-verification-page">
+        <h1 class="certificate-verification-page__title">{{ title }}</h1>
+        <form
+          class="certificate-verification-page__form"
+          novalidate
+          @submit="submit"
+        >
+          <div class="certificate-verification-page__form__input-field">
+            <label for="code">{{ verificationCodeLabel }}</label>
+            <input
+              id="code"
+              v-model="code"
+              :class="[hasCodeRegexError ? 'regex-error' : 'no-regex-error']"
+              type="text"
+              required="true"
+              :pattern="codeRegex"
+              name="verificationCode"
+            />
+            <p
+              v-if="hasCodeRegexError"
+              class="certificate-verification-page__form__input-field__error-paragraph"
```

Comment:
Pareil pour tous les endroits ou on a deux éléments (cad deux `__` ) on peut essayer de nommer simplement l'élément sans faire référence au parent.
```suggestion
              class="certificate-verification-page__error"
```
Ou encore : 
```suggestion
              class="certificate-verification-page__error-for-input"
```
En fait on aurait que 3 questions à se poser pour écrire une classe BEM (si j'ai bien saisie) :
1- dans quel bloc on est ? ici `certificate-verification-page`
2- quel élément je touche ? ici `__error` par exemple
3- est-ce que l'élément est différent / modifié ? ici aucun